### PR TITLE
update iRO files

### PR DESCRIPTION
**Pull request by allanon**
- [ ] QA Review

This pull request was auto-generated by patch_iro.pl: https://gist.github.com/allanon/38620af49d250005acbf

### CODE REVIEW

File: tables/iRO/official/items.txt [hunk 1]
```diff
@@ -6485,7 +6485,7 @@
 17363#January_Groove_Pack#
 17364#March_Groove_Pack#
 17368#Resist_Spell_Power_Pendant_Shadow_Box#
-17369#Rapid_Pendant_Shadow_Box#
+17369#13th_Anniversa-box#
```
Comment (itsrachelfish) [inline tables/iRO/official/items.txt]: The GMs made a typo? :stuck_out_tongue:

**Comment by itsrachelfish** [comment]
Looks good, except for the typo the GMs made? :+1:

**Comment by bahkuteh** [comment]
may i know where the file located please

**Comment by allanon** [comment]
@bahkuteh Are you asking where to find the items list in the official client files? If so, it's in `System/Iteminfo.lub`, which is stored in the `rgz` patch files.

**Pull request merged**